config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -521,10 +526,12 @@ EOF
         # that Kubernetes version, i.e. with the original RBAC rules.
         - name: UPDATE_RBAC_RULES
           value: "true"
-        # docker-in-docker needs privileged mode
-        securityContext:
-          privileged: true
-$(resources_for_kubernetes master   "        ")
+EOF
+    fi
+    if [[ $repo == "external-provisioner" ]]; then
+      cat >>"$base/$repo/$repo-config.yaml" <<EOF
+        - name: VOLUME_MODE_CONVERSION_TESTS
+          value: "true"
```

Comment:
Instead of moving the resource limits, you can put the extra variable inside the EOF block. Something like this should work:
```
        - name: UPDATE_RBAC_RULES
          value: "true"
$(    if [ "$repo" = "external-provisioner" ]; then
cat <<EOF2
        - name: VOLUME_MODE_CONVERSION_TESTS
          value: "true"
EOF2
)
EOF
```

I expect some extra blank line, but IMHO this is still better than the current approach.

Reply:
Thank you for the suggestion! Worked like a charm